README.md

```diff
@@ -1,3 +1,3 @@
 # ToDoListHTML
 
-To work with TodoList, you need to run in the console " npm install " and " npm run build "
+To work with TodoList, you need to run in the console " npm install " then " npm run build ".
```

Comment:
Add explanation that person should open html fileafter this actions, or set up it automatically 
[https://stackoverflow.com/questions/33322053/webpack-launch-browser-automatically](https://stackoverflow.com/questions/33322053/webpack-launch-browser-automatically)